tests/qgis-projects/tests/test_filter_layer_data_by_polygon_for_groups.qgs.cfg

```diff
@@ -0,0 +1,128 @@
+{
```

Comment:
This CFG needs to updated from https://github.com/3liz/lizmap-plugin/pull/355

The configuration was still missing at that time.